networkx/classes/backends.py

```diff
@@ -147,6 +147,7 @@ def wrapper(*args, **kwds):
                     )
         return func(*args, **kwds)
 
+    wrapper._orig_func = func
```

Comment:
All looks good, just some general questions :)

This is to make the dispatch_interface.convert bit work right? With the `NETWORKX_GRAPH_CONVERT` env variable it should automatically be able to do `getattr(nx, item)` in `LoopbackDispatcher.__getattr__` right?

Reply:
This is only to support our loopback testing. Other dispatching backends shouldn't need this.

Before I added `_orig_func`, the tests would go into an infinite loop because the "loopback" function was still decorated and would try the loopback dispatch logic again.